packages/contracts/.gitignore

```diff
@@ -0,0 +1,71 @@
+node_modules
+.env
+
+# Hardhat files
+/cache
+/artifacts
```

Comment:
These lines can be moved to the `.gitignore` file in the root folder and this file can be removed.